qa/CV-for not open position.feature

```diff
@@ -0,0 +1,127 @@
+Feature: Send CV for not open job position  
+Background:
+	Given: User went to the "Send us your CV" form at the down of the page 
+@Happy_pass
+1.1 Scenario: Send CV happy pass
+		Given User entered Elena K in the name field
+		And User entered len--k@yandex.ru in the e-mail field
+		And User checked the checkbox
+		And User attached CV in .pdf file 10 Mb
+		When User press send button
+		Then the CV should be received by recruiters
+		And the message "Thanks for being awesome! We have received your message and will come back to you as soon as we process it" should be on the screen.
+
+1.2 Scenario: Send CV with message happy pass
+		Given User entered name Elena K
+		And User entered len--k@yandex.ru in the e-mail field
+		And User entered 500 symbols in message field
+		And User checked the checkbox
+		And User attached CV in .docx file 10 Mb
+		When User press send button
+		Then the CV should be received by recruiters
+		And the message "Thanks for being awesome! We have received your message and will come back to you as soon as we process it" should be on the screen
+
+@negative_tests
+2.1 Scenario: Send CV with too large message 
+		Given User entered Elena K in the name field
+		And User entered len--k@yandex.ru in the e-mail field
+		And User entered 501 symbols in message field
+		And User checked the checkbox
+		And User attached CV in .docx file 10 Mb
+		When User press send button
+		Then CV should be unsend 
+		And the message "The maximum number of characters is 500" CV should be no the screen
+
+2.2 Scenario: Empty forms, correct attached file
+		Given User tipped no symbols in the name
+		And User tipped no symbols in the e-mail
+		And User attached CV in .pdf file
+		And User checked the checkbox
+		When User press send button
+		Then CV should be unsend
+		And Empty field should be highlighted in red
+		And Messages "This field is required" should be near e-mail and name field
+
+2.3 Scenario: No CV attached
+		Given User entered name Elena K in the name field
+		And User entered len--k@yandex.ru in the e-mail field
+		And User checked the checkbox
+		When User press send button
+		Then CV should be unsend, attached CV button should be highlighted in red
+
+2.4 Scenario: Unchecked checkbox 
+		Given User entered Elena K in the name field
+		And User entered len--k@yandex.ru in the e-mail field
+		And User attached CV in .pdf file
+		When User press send button
+		Then CV should be unsent, checkbox should be highlighted in red
+
+2.5 Scenario: Attached file in incorrect format
+		Given User entered Elena K in the name field
+		And User entered len--k@yandex.ru in the e-mail field
+		And User tried to see file .jpg, .exe, .zip, .fb2, .txt extension 
+		When User opened the folder with nessesary files on the device 
+		Then User should not to see file with inapropriate extension 
```

Comment:
Practically, it's possible to get around this restriction and see all files by clicking "show all files". That's why I would recommend to add one more Scenario, where you will check dialog with appropriate message.